stable/redis-ha/values.yaml

```diff
@@ -26,6 +26,10 @@ serviceAccount:
 ## ref: https://cbonte.github.io/haproxy-dconv/1.9/intro.html
 haproxy:
   enabled: false
+  # Enable if you want a dedicated port in haproxy for redis-slaves
+  readOnlyPort:
```

Comment:
I think renaming this to `slaveEndpoint` or `readOnly` makes more sense. As it's likely we'd want to add stuff in future for read only scenarios.

Reply:
@DandyDeveloper @ssalaues should I rename the same `readOnly`

Reply:
great thought. Changed the value to readOnly